src/Utilities/Providers/FHIRServiceProvider.php

```diff
@@ -115,15 +115,15 @@ public function register()
         // When PatientController needs Adapter Logic point IoC to give the FHIRPatientAdapter
         $this->app->when( 'LibreEHR\FHIR\Http\Controllers\ValuesetController' )
             ->needs( 'LibreEHR\Core\Contracts\BaseAdapterInterface' )
-            ->give( 'LibreEHR\FHIR\Adapters\FHIRValuesetAdapter' );
+            ->give( 'LibreEHR\FHIR\Adapters\FHIRValueSetAdapter' );
 
         // When Adapter needs Repository Logic point IoC to give the PatientRepository
-        $this->app->when( 'LibreEHR\FHIR\Adapters\FHIRValuesetAdapter' )
+        $this->app->when( 'LibreEHR\FHIR\Adapters\FHIRValueSetAdapter' )
```

Comment:
Getting  error
BindingResolutionException in Container.php line 748:
Target [LibreEHR\Core\Contracts\RepositoryInterface] is not instantiable while building [LibreEHR\FHIR\Http\Controllers\ValuesetController, LibreEHR\FHIR\Adapters\FHIRValueSetAdapter].

when commented this call of Repository